packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts

```diff
@@ -28,14 +28,51 @@ import { isGuildBasedChannel } from '../type-guards';
  *
  * @example
  * ```typescript
- * const handler = new PaginatedMessage();
+ * const myPaginatedMessage = new PaginatedMessage();
+ * // Once you have an instance of PaginatedMessage you can call various methods on it to add pages to it.
+ * // For more details see each methods' documentation.
+ *
+ * myPaginatedMessage.addPageEmbed((embed) => {
+ *		embed
+ *			.setColor('#FF0000')
+ *			.setDescription('example description');
+ *
+ *		return embed;
+ * });
+ *
+ * myPaginatedMessage.addPageBuilder((builder) => {
+ *		const embed = new MessageEmbed()
+ *			.setColor('#FF0000')
+ *			.setDescription('example description');
+ *
+ *		return builder
+ *			.setContent('example content')
+ *			.setEmbed(embed);
+ * });
+ *
+ * myPaginatedMessage.addPageContent('Example');
+ *
+ * myPaginatedMessage.run(message)
  * ```
  *
+ * @remark You can also provide a MessageEmbed template. This will be applied to every page.
+ * If a page itself has an embed then the two will be merged into 1, with the data of
+ * the page's own embed taking higher priority over that of the template.
```

Comment:
```suggestion
 * If a page itself has an embed then the two will be merged, with the content of
 * the page's embed taking priority over the template.
```
The nature of merging is that two things will be merged into one, so specifying is redundant and verbose. `{x}'s own {y}` is a redundant structure since `'s` implies ownership, so avoid that in writing. To take priority over something is to assume a position of higher priority, so you do not take higher priority *over* something, but you can have a higher priority when compared to something - I chose the former structure for brevity.